web/modules/custom/itk_activity/js/zipcode.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * @file
+ * Contains javascript to get field_area from field_zipcode using dawa.
+ */
+
+(function ($) {
+  $(document).ready(function () {
+    /**
+     * Update area field with data from dawa.
+     */
+    var updateArea = function updateArea() {
+      var zipcode = $('.js-field-zipcode').val();
```

Comment:
Should we declare var zipcode outside to function so we can reuse it in line 24?

Maybe @cableman has some input?